tests/vmi_multus_test.go

```diff
@@ -45,15 +45,15 @@ const (
 	sriovConfCRD = `{"apiVersion":"k8s.cni.cncf.io/v1","kind":"NetworkAttachmentDefinition","metadata":{"name":"%s","namespace":"%s","annotations":{"k8s.v1.cni.cncf.io/resourceName":"intel.com/sriov"}},"spec":{"config":"{ \"name\": \"sriov\", \"type\": \"sriov\", \"ipam\": { \"type\": \"host-local\", \"subnet\": \"10.1.1.0/24\" } }"}}`
 )
 
-var _ = Describe("Multus Networking", func() {
+var _ = Describe("Multus", func() {
 
 	flag.Parse()
 
 	virtClient, err := kubecli.GetKubevirtClient()
 	tests.PanicOnError(err)
 
-	nodes, err := virtClient.CoreV1().Nodes().List(v13.ListOptions{})
-	tests.PanicOnError(err)
+	nodes := tests.GetAllSchedulableNodes(virtClient)
```

Comment:
looks like you need to put it under `BeforeAll` method or remove
```
Expect(err).ToNot(HaveOccurred(), "Should list compute nodes")
```
from the `GetAllSchedulableNodes` method.

Reply:
Thanks my bad. I update the PR now move this section into the `BeforeAll` I hope the tests will pass now